src/verify-conditions.js

```diff
@@ -47,45 +48,105 @@ const verifyConditions = async (
         );
     }
 
-    // verify that the GITHUB_TOKEN env variable is set
-    // if it isn't set, throw immediately, since verification functions below won't work
-    if (!process.env.GITHUB_TOKEN) {
+    // verify that either `aws` or `github` config option is set with correct info
+    // throw immediately if both are set, or if neither are set
+    if ((!github && !aws) || (github && aws)) {
```

Comment:
I don't if this reads any better, but throwing it out there:

```js
if (Boolean(aws) === Boolean(github)) {

}
```

More succinctly `!aws` would work too (although it inverts the value).

Reply:
thanks for the suggestion.  although I don't love the way it looks now, I think it's easier to understand